### PR TITLE
Improve signup views && ask if over 15

### PR DESCRIPTION
**Pull request by Osamah-learn**
1- 

Screenshot 2021-12-13 at 16 50 26

2-

3- 

Screenshot 2021-11-18 at 13 15 16

4-

Screenshot 2021-11-18 at 13 15 27

5- 

Screenshot 2021-11-18 at 13 15 54

6-

Screenshot 2021-11-18 at 13 16 05

7-

Screenshot 2021-11-18 at 13 16 49

8-

Screenshot 2021-11-18 at 13 16 38

### CODE REVIEW

**Review by ahto — CHANGES_REQUESTED** [review]
it feels like the arrow is not quite right. could there be more air between button text and arrow?

Screenshot_20211123-143426

the spinning gear icon looks little weird. is it too close to the button text?
and the text in the button jumps around when gear icon shows up

https://user-images.githubusercontent.com/59723/143025673-8ffb7b86-0650-44df-8294-32d0c356781b.mp4

File: src/Screens/Onboarding/DisplayName.tsx [hunk 1]
```diff
@@ -43,46 +43,61 @@ const DisplayName = ({ navigation }: Props) => {
...
+              isValidDisplayName ? styles.nextButton : styles.notValidButton
+            }
+            badgeStyle={styles.badgeStyle}
+            onPress={navigateToEmailScreen}
+            messageId="onboarding.displayName.nextButton"
+            badge={require('../images/arrow-right.svg')}
+            testID="onboarding.displayName.nextButton"
+            disabled={!isValidDisplayName}
+          />
+        </RN.View>
```
Comment (pheis) [inline src/Screens/Onboarding/DisplayName.tsx:81]: Could we get rid of this <RN.View> if we give some styles to the Card component?

File: src/Screens/Onboarding/DisplayName.tsx [hunk 2]
```diff
@@ -43,46 +43,61 @@ const DisplayName = ({ navigation }: Props) => {
...
     </OnboardingBackground>
   );
 };
 
 const styles = RN.StyleSheet.create({
   card: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    marginVertical: 20,
```
Comment (pheis) [inline src/Screens/Onboarding/DisplayName.tsx:92]: are we using 20 here? multiples of 8 are best for high DPI stuff. But if design requires 20 then 20 is the best probably.

File: src/Screens/Onboarding/Email.tsx [hunk 3]
```diff
@@ -47,56 +47,79 @@ const Email = ({ navigation }: Props) => {
...
-          testID="onboarding.email.nextButton"
-          disabled={!isValidEmail}
-        />
-        <Button
-          messageId="onboarding.signUp.back"
-          onPress={goBack}
-          noShadow={true}
-          style={{ backgroundColor: colors.lightestGray }}
-        />
+        <RN.View>
```
Comment (pheis) [inline src/Screens/Onboarding/Email.tsx:50]: This View looks kinda excess?

File: src/Screens/Onboarding/DisplayName.tsx [hunk 4]
```diff
@@ -43,46 +43,61 @@ const DisplayName = ({ navigation }: Props) => {
...
-          testID="onboarding.displayName.nextButton"
-          disabled={!isValidDisplayName}
-        />
-        <Button
-          messageId="onboarding.signUp.back"
-          onPress={goBack}
-          noShadow={true}
-          style={{ backgroundColor: colors.lightestGray }}
-        />
+        <RN.View>
```
Comment (pheis) [inline src/Screens/Onboarding/DisplayName.tsx:46]: Does this View do anything? Better to remove it, if it is not necessary.

File: src/Screens/Onboarding/Email.tsx (hunk 3 above)
Comment (Osamah-learn) [reply src/Screens/Onboarding/Email.tsx:50]: its needed , because its a flex container

**Review by idolgov — APPROVED** [review]

**Review by ahto — DISMISSED** [review]

**Review by ahto — APPROVED** [review]

**Pull request merged**